src/tls.c

```diff
@@ -7150,6 +7151,72 @@ static const char* OQS_ID2name(int id)
     }
     return NULL;
 }
+
+typedef struct OqsHybridMapping {
+    int hybrid;
+    int ecc;
+    int oqs;
+} OqsHybridMapping;
+
+static OqsHybridMapping oqs_hybrid_mapping[] = {
+    {.hybrid = WOLFSSL_P256_NTRU_HPS2048509, .ecc = WOLFSSL_ECC_SECP256R1,
+     .oqs = WOLFSSL_NTRU_HPS2048509},
+    {.hybrid = WOLFSSL_P384_NTRU_HPS2048677, .ecc = WOLFSSL_ECC_SECP384R1,
+     .oqs = WOLFSSL_NTRU_HPS2048677},
+    {.hybrid = WOLFSSL_P521_NTRU_HPS4096821, .ecc = WOLFSSL_ECC_SECP521R1,
+     .oqs = WOLFSSL_NTRU_HPS4096821},
+    {.hybrid = WOLFSSL_P384_NTRU_HRSS701,    .ecc = WOLFSSL_ECC_SECP384R1,
+     .oqs = WOLFSSL_NTRU_HRSS701},
+    {.hybrid = WOLFSSL_P256_LIGHTSABER,      .ecc = WOLFSSL_ECC_SECP256R1,
+     .oqs = WOLFSSL_LIGHTSABER},
+    {.hybrid = WOLFSSL_P384_SABER,           .ecc = WOLFSSL_ECC_SECP384R1,
+     .oqs = WOLFSSL_SABER},
+    {.hybrid = WOLFSSL_P521_FIRESABER,       .ecc = WOLFSSL_ECC_SECP521R1,
+     .oqs = WOLFSSL_FIRESABER},
+    {.hybrid = WOLFSSL_P256_KYBER512,        .ecc = WOLFSSL_ECC_SECP256R1,
+     .oqs = WOLFSSL_KYBER512},
+    {.hybrid = WOLFSSL_P384_KYBER768,        .ecc = WOLFSSL_ECC_SECP384R1,
+     .oqs = WOLFSSL_KYBER768},
+    {.hybrid = WOLFSSL_P521_KYBER1024,       .ecc = WOLFSSL_ECC_SECP521R1,
+     .oqs = WOLFSSL_KYBER1024},
+    {.hybrid = WOLFSSL_P256_KYBER90S512,     .ecc = WOLFSSL_ECC_SECP256R1,
+     .oqs = WOLFSSL_KYBER90S512},
+    {.hybrid = WOLFSSL_P384_KYBER90S768,     .ecc = WOLFSSL_ECC_SECP384R1,
+     .oqs = WOLFSSL_KYBER90S768},
+    {.hybrid = WOLFSSL_P521_KYBER90S1024,    .ecc = WOLFSSL_ECC_SECP521R1,
+     .oqs = WOLFSSL_KYBER90S1024},
+    {.hybrid = 0, .ecc = 0, .oqs = 0}
+};
+
+/* This will map an ecc-oqs hybrid group into its ecc group and oqs group.
+ * If it cannot find a mapping then *oqs is set to group. ecc is optional. */
+static void findEccOqs(int *ecc, int *oqs, int group) {
```

Comment:
Function should have explicit start brace.

Reply:
@dgarske , what does "explicit start brace" mean?

Reply:
```
static void findEccOqs(int *ecc, int *oqs, int group)
{
```

Reply:
If it means drop the brace, then done. Please see next commit.